tests/langchain/conftest.py

```diff
@@ -34,6 +36,12 @@ def reset_autolog(reset_autolog_state):
     return
 
 
+@pytest.fixture(autouse=True)
```

Comment:
This is mlflow3 only configuration. We cannot fix the test on the master branch.